apps/docs/content/components/CTAForm.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: CTAForm
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/CTAForm/CTAForm.tsx
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1377%3A30754'
+storybook: '/brand/storybook/?path=/story/components-CTAForm--playground'
+description: Use CTAForm to create a form that is used to collect minimal information from users.
+---
+
+import {
+  CTAForm,
+  InlineLink,
+  FormControl,
+  TextInput,
+  Checkbox,
+} from '@primer/react-brand'
+
+```js
+import {CTAForm} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required size="large" fullWidth>
```

Comment:
I love the composable API 👍. I'm just wondering if some of these critical default values should be baked into the internal logic, while also allowing users to override them if necessary. 

If for example, I remove some of the optional attributes the appearance of the component changes quite dramatically. 

![docs example](https://user-images.githubusercontent.com/13340707/218471345-fb739d05-35c7-4d17-9ded-236530cae900.png)

While we should trust the user to copy the snippet verbatim, some of these feel so crucial to the overarching pattern that the component should be handling them internally for convenience. 

Maybe this is as simple as doing a `find` for a `FormControl` instance in the function for `CTAForm.Input`, and adding default props in.

What do you think?

Reply:
I think that is a great idea! I would imagine setting `fullWidth` and `size='large'` by default is there anything else I am missing there?

Reply:
nope, just those two based on the code snippet. I'd be surprised if `required` wasn't always necessary on the form input too 🤔.

Reply:
I think it's trickier for required, because it's a functional change, vs aesthetic. It may not be immediately clear how to turn off required, if anyone doesn't want it. 

I'm happy to add it in. Can we add a note in the docs to indicate that we set required on the input by default, and that to turn it off they'll need to pass required={false}?